google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1alpha2/SchemaCompactTest.java

```diff
@@ -72,56 +93,763 @@ public Builder toBuilder() {
           }
         };
     when(mockBigqueryTable.getDefinition()).thenReturn(definition);
+  }
+
+  @Test
+  public void testSuccess() throws Exception {
+    customizeSchema(
+        Schema.of(
+            Field.newBuilder("Foo", LegacySQLTypeName.STRING)
+                .setMode(Field.Mode.NULLABLE)
+                .build()));
```

Comment:
Not in scope of this CL. Once we support schema update in writeapi, we could change the SchemaCompact to be built with a fixed BQ Schema, and allow it to be tested against multiple proto inputs. You may add a TODO somewhere on the class ctor to note this. The backend isn't ready for this now.

Reply:
Is this for the customizeSchema() function? I'm just using this to customize the BQ schema for the tests.

Reply:
Yeah, we can discuss this later.